packages/core/types/tests.tsx

```diff
@@ -47,6 +47,13 @@ const ComponentWithCache = withEmotionCache((_props: {}, context) => {
     }}
   />
 </div>
+;<Global
```

Comment:
GitHub (and my editor emacs) cannot syntax highlight all these things.... Oh my prettier...